### PR TITLE
Minor UI/Wording changes

### LINKED ISSUE #65: Title and site name should be the Participation Hub vs. Contribution Hub
Firefox OS Contribution Hub -> Firefox OS Participation Hub

### LINKED ISSUE #66: Logo / Title Spacing Issues
Feedback from @geroter :

The logo and spacing with Contribution Hub doesn’t look quite right. Maybe create a logo mark using the Firefox OS- Participation type of logo for the site (Firefox OS on one line, next line underneath “Participation Hub”)

### LINKED ISSUE #67: Change Call to Action
The call to action shouldn’t be “Hack it on your device” but instead “Install it on your device” or “Get it on your device”.

### LINKED ISSUE #68: Nav Bar Text Changes
Change the menu items:
- “Hack” to “Contribute”
- “Foxfooding” to “Foxfood”

### LINKED ISSUE #70: Home Page Contribution Section Spacing
Icons on the bottom right are too closely spaced to the text.

### LINKED ISSUE #72: Text for Dashboard Page
From @geroter ....

definitely need explanation text at the top. My suggestion:

Help make Firefox OS better. Contribute to prioritizing features or defects by voting for those you most want to see worked on. Report a feature or defect by clicking the ‘+’ button on the right.

### LINKED ISSUE #73: Add Images to Foxfooding Pages
About and FAQ should have images, similar to the other flat pages.

### PR DESCRIPTION
**Pull request by comzeradd**
- More generous space on home minor CTAs (Fixes #70)
- Change title to Participation Hub (Fixes #65)
- Change wording to navbar menus (Fixes #68)
- Change wording on main CTA (Fixes #67)
- Change foxfooding flatpage layout (Fixes #73)
- Add info sign/text on Dashboard (Fixes #72)
- Split title (Fixes #66)

### CODE REVIEW

**Comment by comzeradd** [comment]
@johngian r?

File: feedthefox/dashboard/templates/jinja2/dashboard.html [hunk 1]
```diff
@@ -5,7 +5,10 @@
 {% endblock %}
 
 {% block content %}
-  <h3>Dashboard</h3>
+  <h3 class="dashboard-title">Dashboard</h3>
+  <i class="fa fa-info-circle"
+     title="Help make Firefox OS better. Contribute to prioritizing features or defects by voting for those you most want to see worked on. Report a feature or defect by clicking the ‘+’ button on the right.">
```
Comment (johngian) [inline feedthefox/dashboard/templates/jinja2/dashboard.html:10]: This element doesn't work locally. I am getting this error:
`Empty string passed to getElementById().`

**Comment by johngian** [comment]
Code looks OK. r+wc

File: feedthefox/dashboard/templates/jinja2/dashboard.html (hunk 1 above)
Comment (comzeradd) [inline feedthefox/dashboard/templates/jinja2/dashboard.html:10]: It works here. I get this error when I actually press the '+' button (although it still works). But this is from an old merge, so I investigate further.

**Pull request merged**